pkg/swarm/swarm.go

```diff
@@ -37,6 +37,12 @@ var (
 	ErrInvalidChunk = errors.New("invalid chunk")
 )
 
+var (
+	// Ethereum Address of the for Dispersed Replicas writes
```

Comment:
Seems like some bad grammar in the comment?

Reply:
Thanks for the fast catch!